src/Microsoft.ML.TensorFlow/TensorFlow/TensorFlowModelContext.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Data;
+
+namespace Microsoft.ML.Transforms.TensorFlow
```

Comment:
I would probably put it into same namespace as tensorflow transform, i.e. Microsoft.ML.Transforms.

Also requires update in documentation.
Would be nice to explain user in which case he need to use simple constructor and in which this Context object.